core/src/main/java/io/questdb/griffin/engine/functions/groupby/StdDevPopDoubleGroupByFunction.java

```diff
@@ -40,61 +35,18 @@
  *
  * @see <a href="https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Welford's_online_algorithm">Welford's algorithm</a>
  */
-public class StdDevPopDoubleGroupByFunction extends DoubleFunction implements GroupByFunction, UnaryFunction {
-    private final Function arg;
-    private int valueIndex;
+public class StdDevPopDoubleGroupByFunction extends AbstractStatAggregatorGroupByFunction {
```

Comment:
nit: we could remove `Double` from the function name since all of these functions can only return double values, i.e. `StdDevPopGroupByFunction` instead of `StdDevPopDoubleGroupByFunction`. The same suggestion applies to other modified functions.